src/components/ProjectIndex/ProjectIndex.js

```diff
@@ -37,11 +37,7 @@ const ProjectIndex = (props) => {
           ButtonIcon={PlusIcon}
         />
       </ProjectIndexHeader>
-      <div className='main-container'>
-        {projectList.map((project, i) => (
-          <ProjectListItem project={project} user={user} key={i} />
-        ))}
-      </div>
+      <ProjectListTable projects={projectList} user={user}/>
```

Comment:
I'm wondering, is there a reason to pass the `projectList` and `user` into this rather than extracting them within the `ProjectListTable` component itself (via `useSelector`)`?

Reply:
No reason! Just me being stupid :upside_down_face:
Done in [cf2755e](https://github.com/RaspberryPiFoundation/editor-ui/pull/317/commits/cf2755e4425b084d91f55656a7549d81e11c8281)